### ISSUE TITLE
Loss of D-net

### ISSUE THREAD

**Issue by yxt7979** [baegwangbin/MaGNet#9]
Hi, Excellent work!
 I found the loss of D-Net is ` nll = (torch.square(mu - gt_depth) / (2 * var)) + (0.5 * torch.log(var))` in [here](https://github.com/baegwangbin/MaGNet/blob/master/utils/losses.py#L21).
I have two questions below:
1. will the loss turns negative as log(1e-5) < 0 (take 1e-5 for example)
2. how long does the training step of D-net take for each dataset?

Thank you for your reply!!

**Comment by baegwangbin** [comment]
Hi, thanks for your interest.

1. Yes, the loss becomes negative and this is not a problem.
2. We trained the networks for 5 epochs for ScanNet and 10 epochs for KITTI.

**Comment by yxt7979** [comment]
Thanks!
